content/cfp.html

```diff
@@ -5,10 +5,12 @@
 callout_big_2: ...you
 ---
 
-<p>To submit a proposal to run a session at PyCon UK 2016, please complete <strong><a href="https://docs.google.com/forms/d/1IkIuN13wwDl1gH1L00HQ6UaX-3PWd3uFHuHvCmEJYGA/viewform">the proposal form</a></strong>.  Submissions are open until the end of June 28th (BST).</p>
-
 <p>This is a <strong>community conference</strong>. You’re coming to the event, and that makes <strong>you</strong> part of the community and makes this <strong>your</strong> conference. If you’ve done something that you think other people might be interested in, or that you think they should know about, <em>we would love to know about it</em>.</p>
 
+<a href="/tickets" class="btn small">Submit a proposal</a>
```

Comment:
Could we centralise this button?  I think that

    <div align="center"><a href="/tickets" class="btn small">Submit a proposal</a></div>

looks a little better to me.

What do you think?